ui/helpers/utils/util.js

```diff
@@ -516,3 +517,20 @@ export function coinTypeToProtocolName(coinType) {
 export function isNullish(value) {
   return value === null || value === undefined;
 }
+
+/**
+ * The method escape RTL character in string
+ *
+ * @param {string} str
+ * @returns {string} escaped string
```

Comment:
```suggestion
 * @returns {(string|*)} escaped string or original param value
```

`if (!str)`, then this function may return something other than a string